firefox-ios/Client/Frontend/Browser/BrowserViewController/State/BrowserViewControllerState.swift

```diff
@@ -12,6 +12,7 @@ struct BrowserViewControllerState: ScreenState, Equatable {
         case home
         case back
         case forward
+        case backForwardList
```

Comment:
Originally I wanted the `NavigationType` to be in browser navigation only. If we stick with that then showing the back/forward list would not fit in there and should rather be handled like showing the QR code reader.